single_offer_router/src/lib.rs

```diff
@@ -87,16 +88,16 @@ pub fn offer_salt(
     match admin {
         Identifier::Contract(a) => salt_bin.append(&a.clone().into()),
         Identifier::Ed25519(a) => salt_bin.append(&a.clone().into()),
-        Identifier::Account(a) => salt_bin.append(&a.clone().into()),
+        Identifier::Account(a) => salt_bin.append(&a.serialize(&e)),
```

Comment:
This appears to be the only functional change, but looks fine to me. The bytes used in the salt for an account is now the serialized XDR of an AccountId, rather than the ed25519 public key. cc @sisuresh

Reply:
Yeah this is fine. Once this is merged, I'm going to rename `offer_salt` to `offer_key` because my usage of salt here doesn't make sense.

Reply:
Merged!